### PR TITLE
changed creating loan policy

### PR DESCRIPTION
**Pull request by TetianaParanich**
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1366/allure/

### CODE REVIEW

**Review by MaksimTerekhov — APPROVED** [review]

**Review by re-jas — APPROVED** [review]

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js [hunk 1]
```diff
@@ -12,42 +10,49 @@ import Checkout from '../../../support/fragments/checkout/checkout';
...
+    InventoryHoldings.getHoldingSources({ limit: 1 }).then((source) => {
+      holdingsSourceId = source.id;
+    });
     cy.getInstanceTypes({ limit: 1 });
     cy.getLocations({ limit: 1 });
     cy.getHoldingTypes({ limit: 1 });
     cy.createLoanType({ name: loanTypeName });
+    ServicePoints.getViaApi()
+      .then((res) => {
+        servicePointId = res[0].id;
```
Comment (penfriend) [inline cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:52]: are you sure that this service point , that you have got in this method, will exist till the end of this test?
please create a new service point

Comment (TetianaParanich) [reply cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js]: Done

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js [hunk 2]
```diff
@@ -12,42 +11,46 @@ import Checkout from '../../../support/fragments/checkout/checkout';
...
     cy.getAdminToken();
-    cy.getMaterialTypes({ limit: 1 });
+    cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+      materialType = materialTypes;
+    });
+    InventoryHoldings.getHoldingSources({ limit: 1 }).then((source) => {
+      holdingsSourceId = source.id;
+    });
     cy.getInstanceTypes({ limit: 1 });
     cy.getLocations({ limit: 1 });
```
Comment (penfriend) [inline cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:49]: please create also unique location

Comment (TetianaParanich) [reply cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js]: Done

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js [hunk 3]
```diff
@@ -12,42 +11,51 @@ import Checkout from '../../../support/fragments/checkout/checkout';
...
+import LoanPolicyActions from '../../../support/fragments/circulation/loan-policy';
+import CirculationRules from '../../../support/fragments/circulation/circulation-rules';
+import Location from '../../../support/fragments/settings/tenant/locations/newLocation';
 
 describe('ui-users-loans: renewal failure because loan has reached maximum renewals', () => {
   const loanTypeName = `autotest_loan_type${getRandomPostfix()}`;
   const newFirstItemData = getNewItem();
   const newSecondItemData = getNewItem();
+  const servicePoint = ServicePoints.getDefaultServicePointWithPickUpLocation('autotest', uuid());
+  const servicePointId = servicePoint.id;
```
Comment (penfriend) [inline cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:30]: Why do you need this variable? 
You can actually use servicePoint.id

Comment (TetianaParanich) [reply cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js]: Done

Comment (TetianaParanich) [reply cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js]: Done

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js [hunk 4]
```diff
@@ -12,42 +11,50 @@ import Checkout from '../../../support/fragments/checkout/checkout';
...
+    cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+      materialType = materialTypes;
+    });
+    InventoryHoldings.getHoldingSources({ limit: 1 }).then((source) => {
+      holdingsSourceId = source.id;
+    });
     cy.getInstanceTypes({ limit: 1 });
-    cy.getLocations({ limit: 1 });
     cy.getHoldingTypes({ limit: 1 });
     cy.createLoanType({ name: loanTypeName });
```
Comment (penfriend) [inline cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:51]: Let's create a variable . This variable will save the result of the response of the function cy.createLoanType({ name: loanTypeName });

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js [hunk 5]
```diff
@@ -215,15 +211,19 @@ describe('ui-users-loans: renewal failure because loan has reached maximum renew
...
       InventoryInstance.deleteInstanceViaApi(secondInstanceIds.instanceId);
     });
-
-    cy.deleteLoanPolicy(Cypress.env(CY_ENV.LOAN_POLICY).id);
-    RequestPolicy.deleteViaApi(Cypress.env(CY_ENV.REQUEST_POLICY).id);
-    LostItemFeePolicy.deleteViaApi(Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY).id);
-    OverdueFinePolicy.deleteViaApi(Cypress.env(CY_ENV.OVERDUE_FINE_POLICY).id);
-    NoticePolicy.deleteViaApi(Cypress.env(CY_ENV.NOTICE_POLICY).id);
-
+    cy.deleteLoanPolicy(loanPolicy.id);
```
Comment (penfriend) [inline cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:214]: please delete the loan type that you create in precondition

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js]: Done

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js (hunk 5 above)
Comment (TetianaParanich) [reply cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:216]: Done

**Pull request merged**